### PR TITLE
refactor the img tag

### LINKED ISSUE #7094: img tag refactor in CONTRIBUTING.md Sec 1.3 - second instance
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started) and attend an onboarding session.
2. Before you claim or start working on an issue, please make sure you have read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/7f0c132c96f71230b8935759e1f8711ccb340c0f/CONTRIBUTING.md).

### Overview
We want to change an img HTML tag ending with a slash () to an img tag without an ending slash () so that the codebase is consistent with how we use img HTML tags.

### Action Items
- [x] In your local IDE, open the file `CONTRIBUTING.md`
- [x] In Section 1.3 change
```
  <img src="https://user-images.githubusercontent.com/21162229/137045294-3d46b28c-edbb-410c-98f1-13940ecc5c5f.jpg" />       

```
to
```
   <img src="https://user-images.githubusercontent.com/21162229/137045294-3d46b28c-edbb-410c-98f1-13940ecc5c5f.jpg">      

```
**- [ ] The file may contain more than one instance of an img tag with a closing slash.  Only change the single instance specified above.  Do not leave any blank spaces before the closing bracket.** 
- [x] Changes to CONTRIBUTING.md cannot be tested locally, rather they must be tested after pushing the issue branch to your fork of the repository.  Push your issue branch in the usual manner, but before creating the Pull Request, check your updates using this test URL (also store the test URL for use in a later step)
```
https://github.com/[REPLACE WITH GITHUB HANDLE]/website/blob/[REPLACE WITH NAME OF ISSUE BRANCH]/CONTRIBUTING.md
```
   (for example: https://github.com/roslynwythe/website/blob/test-CONTRIB/CONTRIBUTING.md)
- [x] Create a pull request with your changes.  In the Pull Request, after the "Why did you make the changes" section, add this line to help reviewers, replacing the text in brackets (and the brackets) with the test URL from the previous Action Item.  
```
For Reviewers: Do not test changes locally, rather test changes at [REPLACE WITH TEST URL]
```

### Merge Team
- [x] After closing this issue, check off the dependency in #6453.  If all dependecies have been completed, close #6453 and check off its dependency in #4362.  If all dependencies have been completed in #4362, close it.  

### Resources/Instructions
1. Webpage(s): https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md
2. This issue is part of #6453

### PR DESCRIPTION
**Pull request by blaycoder**
Fixes #7094 

### What changes did you make?
  - I changed an img HTML tag ending with a slash () to an img tag without an ending slash ().

### Why did you make the changes (we will use this info to test)?
  - So that the codebase is consistent with how we use img HTML tags.
  - https://github.com/blaycoder/website/blob/refactor-img-tag-7094/CONTRIBUTING.md

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)
Changed the  img HTML tag ending with a slash () to an img tag without an ending slash in the CONTRIBUTING.md. No visual changes to the website.

### CODE REVIEW

**Review by daras-cu — CHANGES_REQUESTED** [review]
Hi @blaycoder, thanks for working on this issue!

- The branches are set up correctly and the original issue is linked in your PR
- You explained the changes you made and why
- No visual changes to the website and CONTRIBUTING.md displays correctly on your branch

A couple minor things to change:
- It looks like when you edited the img tag you also added a space at the beginning of the line. This should be removed so that the indentation is consistent with the preceding lines
- I would probably make your PR title and description a little more specific by including the file and section the changes were made in, similar to the title of the original issue (#7094)

Overall good work on this issue, let me know in a comment or re-request a review when you've made adjustments.

**Comment by blaycoder** [comment]
Hi @daras-cu 
Thank you so much for the observation, I have made the changes and equally pushed to the branch. 
Kindly review

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -118,7 +118,7 @@ You can fork the hackforla/website repository by clicking <a href="https://githu
   <p><strong>First, you find the `fork` button on the top right hand side of the [Hack for LA website repo](https://github.com/hackforla/website)</strong></h4>
   <img src="https://user-images.githubusercontent.com/21162229/137044762-c80f18e8-b785-48fb-a286-81c1117c0178.jpg" />
   <p><strong>Next, a modal may open and request where you want to fork this website repo. Please click on your avatar or your GitHub username</strong></p>
-  <img src="https://user-images.githubusercontent.com/21162229/137045294-3d46b28c-edbb-410c-98f1-13940ecc5c5f.jpg" />
+  <img src="https://user-images.githubusercontent.com/21162229/137045294-3d46b28c-edbb-410c-98f1-13940ecc5c5f.jpg">      
 </details><br>
 
 **Note:** It should create a URL that looks like the following -> `https://github.com/<your_GitHub_user_name>/website`.
```
Comment (mSharifHub) [inline CONTRIBUTING.md:124]: If this comment belongs to the `` then would be more clear by commenting on top of the line of code.

**Review by mSharifHub — COMMENTED** [review]
I have not approved but I pointing out a readability issue to what line of code node is referring too. Is just an observation. Overall,  I do not see no issues.

**Review by daras-cu — APPROVED** [review]
Hi @blaycoder, thanks for adjusting the whitespace. The changes look good in your branch, thanks for working on this issue!

**Comment by vorleakyek** [comment]
Review ETA: 8/29   
Availability: M-F evening

**Review by vorleakyek — APPROVED** [review]
Nice work! It looks correct. 

Verified that the image is showing up with this testing environment.  
https://github.com/blaycoder/website/blob/refactor-img-tag-7094/CONTRIBUTING.md

**Pull request merged**